streaming-mqtt/README.md

```diff
@@ -52,12 +52,14 @@ this actor can be configured to handle failures, etc.
 
     val lines = MQTTUtils.createStream(ssc, brokerUrl, topic)
     val lines = MQTTUtils.createPairedStream(ssc, brokerUrl, topic)
+    val lines = MQTTUtils.createPairedByteArrayStreamStream(ssc, brokerUrl, topic)
```

Comment:
In MQTTUtils class. This is done exactly in same way as createPairedStream with implementation in MQTTPairedByteArrayInputDStream.scala